src/FakeItEasy/ArgumentConstraintManagerExtensions.cs

```diff
@@ -77,7 +77,11 @@ public static class ArgumentConstraintManagerExtensions
         /// <returns>A dummy argument value.</returns>
         public static string Contains(this IArgumentConstraintManager<string> manager, string value)
         {
+#if FEATURE_STRING_CONTAINS_COMPARISONTYPE
+            return manager.NullCheckedMatches(x => x.Contains(value, StringComparison.CurrentCulture), x => x.Write("string that contains ").WriteArgumentValue(value));
+#else
             return manager.NullCheckedMatches(x => x.Contains(value), x => x.Write("string that contains ").WriteArgumentValue(value));
+#endif

```

Comment:
This fixes a [CA1307](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1307) warning that appeared when I added the netstandard2.1, because new overloads of `String.Contains` with a `StringComparison` parameter have been added